npm-shrinkwrap.json

```diff
@@ -2,11 +2,65 @@
   "name": "etcher",
   "version": "1.0.0-beta.18",
   "dependencies": {
+    "abbrev": {
```

Comment:
There is an NPM bug I've been hitting, when development dependencies get added (this seems to be the case). Since `angular-bind-html-compile-ci-dev` has no dependencies, I guess the easy way to fix is to delete every addition except for that module?